src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -2645,6 +2652,12 @@ private void EmitSizeOfExpression(BoundSizeOfOperator boundSizeOfOperator)
             EmitSymbolToken(type, boundSizeOfOperator.SourceType.Syntax);
         }
 
+        private void EmitMethodTokenExpression(BoundMethodToken node)
+        {
+            _builder.EmitOpCode(ILOpCode.LoadMethodTokenAsInteger_i4);
```

Comment:
@tmat has suggested using ldtoken with the high bit set in the fake token value, and rewriting the instruction to be a ldc during metadata emission when the high bit is set. I'm not sure which approach is easier to understand. Opinons?